### DISCUSSION TITLE
Unable to edit system:deployer role

### DISCUSSION THREAD

**Discussion by K1kc4** [okd-project/okd#1721] [Q&A] [answered]
Since 4.13 update i am unable to edit "system:deployer" role, its allways going back to default.

I have tried adding annotation `rbac.authorization.kubernetes.io/autoupdate: "false"` , but nothing changes.

Does anyone else has the same problem ?

Kind regards, Yordan

**Comment by vrutkovs** [comment]
This role is created by internal controllers and cannot be modified. Why would you want to modify its permissions instead of creating a new role?

**Reply by K1kc4** [reply]
Actually we were able to do it before. Its easier way, because we dont need to create RoleBinding in every project,its done automatically.

Is there a way to make a rolebinding, that is gonna be bound with deployer sa in every namespace, same as system:deployer?

**Reply by tyronewilsonfh** [reply, marked as answer]
Follow https://docs.okd.io/latest/applications/projects/configuring-project-creation.html#modifying-template-for-new-projects_configuring-project-creation to modify the template used for creation of each project. Add the rolebinding there and it will be created on creation of a new project, you need to specifically create a project for this to work, not a namespace.

**Reply by K1kc4** [reply]
It did the job, thanks for help.
